### PR TITLE
ARROW-6204: [GLib] Add garrow_array_is_in_chunked_array()

### PR DESCRIPTION
**Pull request by shiro615**
This is follow-up of https://github.com/apache/arrow/pull/5047#issuecomment-520103706.

### CODE REVIEW

File: c_glib/test/test-is-in.rb [hunk 1]
```diff
@@ -18,31 +18,46 @@
...
+  sub_test_case("ChunkedArray") do
+    def test_no_null
+      left_array = build_int16_array([1, 0, 1, 2])
+      chunks = [
+        build_int16_array([1, 0]),
+        build_int16_array([1, 0, 3])
+      ]
+      right_chunked_array = Arrow::ChunkedArray.new(chunks)
+      assert_equal(build_boolean_array([true, true, true, false]),
+                   left_array.is_in(right_chunked_array))
```
Comment (kou) [inline c_glib/test/test-is-in.rb]: `is_in` -> `is_in_chunked_array`

Comment (shiro615) [reply c_glib/test/test-is-in.rb]: Oh... Thank you for pointing out.

Comment (shiro615) [reply c_glib/test/test-is-in.rb]: I've fixed this.

File: c_glib/arrow-glib/compute.cpp [hunk 2]
```diff
@@ -1440,7 +1441,43 @@ garrow_array_is_in(GArrowArray *left,
...
+
+/**
+ * garrow_array_is_in_chunked_array:
+ * @left: A left hand side #GArrowArray.
+ * @right: A right hand side #GArrowChunkedArray.
+ * @error: (nullable): Return location for a #GError or %NULL.
+ *
+ * Returns: (nullable) (transfer full): The #GArrowBooleanArray
+ *   showing whether each element in the left array is contained
+ *   in right array.
```
Comment (kou) [inline c_glib/arrow-glib/compute.cpp]: "array" -> "chunked array".

File: c_glib/test/test-is-in.rb [hunk 3]
```diff
@@ -18,31 +18,79 @@
...
+      chunks = [
+        build_int16_array([2, 0, 3]),
+        build_int16_array([3, 0, 2, 2])
+      ]
+      right_chunked_array = Arrow::ChunkedArray.new(chunks)
+      assert_equal(build_boolean_array([false, true, nil, true]),
+                   left_array.is_in_chunked_array(right_chunked_array))
+    end
+
+    def test_null_in_right_array
```
Comment (kou) [inline c_glib/test/test-is-in.rb]: `right_chunked_array` or just `right`.

File: c_glib/test/test-is-in.rb [hunk 4]
```diff
@@ -18,31 +18,79 @@
...
+      chunks = [
+        build_int16_array([2, 0, nil, 2, 0]),
+        build_int16_array([2, 3, nil])
+      ]
+      right_chunked_array = Arrow::ChunkedArray.new(chunks)
+      assert_equal(build_boolean_array([false, true, false, true]),
+                   left_array.is_in_chunked_array(right_chunked_array))
+    end
+
+    def test_null_in_both_arrays
```
Comment (kou) [inline c_glib/test/test-is-in.rb]: How about just using `test_null_in_both`?

**Review by kou — CHANGES_REQUESTED** [review]
Could you check my comments?

File: c_glib/arrow-glib/compute.cpp (hunk 2 above)
Comment (shiro615) [reply c_glib/arrow-glib/compute.cpp]: I've fixed it.

File: c_glib/test/test-is-in.rb (hunk 3 above)
Comment (shiro615) [reply c_glib/test/test-is-in.rb]: I've used `right`.

File: c_glib/test/test-is-in.rb (hunk 4 above)
Comment (shiro615) [reply c_glib/test/test-is-in.rb]: I've used it.

**Review by kou — APPROVED** [review]
+1
Thanks!

**Pull request closed without merging**